base/abstractarray.jl

```diff
@@ -2009,6 +2009,24 @@ function (==)(A::AbstractArray, B::AbstractArray)
     return anymissing ? missing : true
 end
 
+"""
+    <(A::AbstractVector, B::AbstractVector)
```

Comment:
Maybe use `x` and `y` for consistency with the existing docstring for `<`.

Reply:
I used `A` and `B` for consistency with `isless` above. Should I change it?

Reply:
No idea. :-/

Reply:
Let us see what @JeffBezanson says given https://github.com/JuliaLang/julia/commit/ada8b0640d953dd8f56798f2b616dd2f79493e64 :)